crates/cheatcodes/spec/src/vm.rs

```diff
@@ -1077,6 +1077,22 @@ interface Vm {
     #[cheatcode(group = String)]
     function parseBool(string calldata stringifiedValue) external pure returns (bool parsedValue);
 
+    /// Converts the given `string` value to Lowercase.
+    #[cheatcode(group = String)]
+    function toLowercase(string calldata stringifiedValue) external pure returns (string memory stringifiedValue);
+    /// Converts the given `string` value to Uppercase.
+    #[cheatcode(group = String)]
+    function toUppercase(string calldata stringifiedValue) external pure returns (string memory stringifiedValue);
+    /// Trims leading and trailing whitespace from the given `string` value.
+    #[cheatcode(group = String)]
+    function trim(string calldata stringifiedValue) external pure returns (string memory stringifiedValue);
+    /// Replaces occurrences of `from` in the given `string` with `to`.
+    #[cheatcode(group = String)]
+    function replace(string calldata stringifiedValue, string calldata from, string calldata to) external pure returns (string memory stringifiedValue);
```

Comment:
Same as https://github.com/foundry-rs/foundry/pull/6891/files#r1465480622
These are not stringifed values, but just strings, so `s` or `str` is more suitable